src/Backend/Modules/Faq/Actions/Edit.php

```diff
@@ -108,12 +108,13 @@ private function validateForm(): void
                 // build item
                 $item = [];
                 $item['id'] = $this->id;
-                $item['meta_id'] = $this->meta->save(true);
+                $item['meta'] = $this->meta;
```

Comment:
Don't change anything in the actions, only in the model.

Reply:
Hmm, how do you handle the meta then?

Reply:
Fixed 232348709